Formula/llvm.rb

```diff
@@ -136,16 +144,23 @@ def install
     builtins_cmake_args = []
 
     # Skip the PGO build on HEAD installs or non-bottle source builds
-    pgo_build = build.stable? && build.bottle?
+    # pgo_build = build.stable? && build.bottle?
+    pgo_build = false # deactivate while testing
 
     if OS.mac?
       args << "-DLLVM_BUILD_LLVM_C_DYLIB=ON"
       args << "-DLLVM_ENABLE_LIBCXX=ON"
+      args << "-DLIBCXX_INSTALL_LIBRARY_DIR=#{lib}/c++"
+      args << "-DLIBCXXABI_INSTALL_LIBRARY_DIR=#{lib}/c++"
       args << "-DDEFAULT_SYSROOT=#{macos_sdk}" if macos_sdk
-      runtimes_cmake_args << "-DCMAKE_INSTALL_RPATH=#{rpath}"
+      runtimes_cmake_args << "-DCMAKE_INSTALL_RPATH=#{loader_path}"
+      # Disable builds for OSes not supported by the CLT SDK.
+      builtins_cmake_args += %w[I WATCH TV].map { |os| "-DCOMPILER_RT_ENABLE_#{os}OS=OFF" }
     else
       ENV.append_to_cflags "-fpermissive -Wno-free-nonheap-object"
 
+      # Enabling libxml2 results in linkage with icu4c, which is awful.
+      args << "-DLLVM_ENABLE_LIBXML2=OFF"
```

Comment:
How? This doesn't seem to be the case with 14.0.6_1, built after libxml2 depended on icu4c?

Reply:
I'll look into it, but I could swear I spotted LLVM linkage with `icu4c` at #98809. See https://github.com/Homebrew/homebrew-core/pull/98809#issuecomment-1202064745.

`libxml2` doesn't seem to enable anything useful in LLVM anyway (it seems to be designed primarily for Windows), so it seems better to just avoid having to rev bump this in case `libxml2` bumps their SO version.

Reply:
I'm fine with this if we take a "one less dependency" angle, though the comment would need changing if so.